lib/landing/landing.dart

```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-
```

Comment:
Let's minimize including these kinds of changes (formatting lines/optimizing code feature in Android Studio/IntelliJ should be run on a file before it's added to a commit)